src/pages/content-edit/head.js

```diff
@@ -3,261 +3,431 @@ import format from "date-fns/format";
+                    m("use", { href : icons + "#save" })
                 ),
-                status === "draft" ?
-                    null :
-                    m("button", {
-                            // Attrs
-                            class    : css.unpublish,
-                            title    : "Unpublish immediately",
-                            disabled : locked || null,
-
-                            // Events
-                            onclick : ctrl.unpublish
-                        },
-                        m("svg", { class : css.icon },
-                            m("use", { href : icons + "#remove" })
-                        ),
-                        "Unpublish"
-                    )
+                "Save"
             )
-        ),
-        ctrl.schedule ? m("div", { class : css.details },
+        );
+    }
+
+
+    function mScheduleButton() {
+        return m("button", {
+                // Attrs
+                class : css.schedule,
+                title : "Schedule a publish",
+
+                // Events
+                onclick : ctrl.toggle.bind(null, undefined)
+            },
+            m("svg", { class : css.onlyIcon },
+                m("use", { href : icons + "#schedule" })
+            )
+        );
+    }
+
+    function mPublishButton() {
+        var isDisabled = false;
+
+        // TODO Better implementation.
+        // if(ctrl.start.ts && isPast(ctrl.start.ts)) {
+        //     isDisabled = true;
+        // }
+
+        return m("button", {
+                // Attrs
+                class    : css.publish,
+                title    : future ? "Schedule publish" : "Already published",
+                disabled : locked || isDisabled || null,
+
+                // Events
+                onclick : ctrl.publish
+            },
+            m("svg", { class : css.icon },
+                m("use", { href : icons + (future ? "#schedule" : "#publish") })
+            ),
+            future ? "Schedule" : "Publish"
+        );
+    }
+
+    function mUnpublishButton() {
+        var isDisabled = false;
+
+        if(status === "draft") {
+            return null;
+        }
+
+        // TODO Better implementation.
+        // if(ctrl.end.ts && isPast(ctrl.end.ts)) {
+        //     isDisabled = true;
+        // }
+
+        return m("button", {
+                // Attrs
+                class    : css.unpublish,
+                title    : wasUnpublished ? "Already unpublished" : "Unpublish immediately",
+                disabled : locked || isDisabled || null,
+
+                // Events
+                onclick : ctrl.unpublish
+            },
+            m("svg", { class : css.icon },
+                m("use", { href : icons + "#remove" })
+            ),
+            "Unpublish"
+        );
+    }
+
+    function scheduleInput(id, type, section, field) {
+        return m("input", {
+            class : css.date + ( ctrl.invalidDates ? " " + css.invalidDate : ""),
```

Comment:
done, this was due to lack of experience with modular css and its best practices